lang/en/texts/index-pro.html

```diff
@@ -0,0 +1,39 @@
+<h1>Open Food Facts for producers</h1>
+
+<div class="row hide-when-logged-in">
+
+<div class="medium-12 columns">
+
+<p>A completely free platform to allow manufacturers to easily manage the data and photos of their products on Open Food Facts.</p>
+
+<h2>Make your products visible in more than 100 apps and services</h2>/
+
+<p>Publish your product data on Open Food Facts and participate in the global movement for transparency on food products.</p>
+
+<ul>
+	<li>Your data is complete, correct and up-to-date.</li>
```

Comment:
Is this something we want to make dynamic?

Reply:
Dynamic?

Reply:
>`<li>Your data is complete, correct and up-to-date.</li>`

What if it is not?

Reply:
Well that's the goal of the platform: they give us their data, and that way it is more complete, correct and up-to-date. Hopefully. If it's not the case (bad or outdated data), we will let them know.